### PR TITLE
Add vanilla os and aeon setup instructions

### LINKED ISSUE #3699: Add Vanilla OS and Aeon setup instructions
[Vanilla OS](https://vanillaos.org) released it's first stable release in the 2.x branch.
[Aeon](https://aeondesktop.github.io) is currently RC3.

Both distros heavily promote the usage of flatpak as the preferred way of installing apps.
Even though they don't require any setup IMHO it would be nice to mirror this information on the setup page.

### PR DESCRIPTION
**Pull request by twihno**
fixes #3699 

@mirkobrombin @sysrich What's your opinion on this description? As mentioned in #3699 I think it would be nice to explicitly state that there is no setup needed for using flatpak/flathub in these distros.

Should it be explicitly mentioned that flathub is the preconfigured remote?

### CODE REVIEW

**Comment by mirkobrombin** [comment]
Speaking for Vanilla OS here. Ase you said, both Flatpak and Flathub are pre-configured in Vanilla OS. But I would also mention the steps to install it in case something went wrong. We documented it [here](https://docs.vanillaos.org/handbook/en/install-flatpaks).

Also thanks for the PR.

**Review by kbdharun — APPROVED** [review]
Thanks for documenting Vanilla OS, LGTM after Mirko's suggestion.

---

If possible can you include a smaller/lighter version of our logo (will request one from @mirkobrombin) since the other distro logo image assets are much smaller and lightweight.

**Comment by razzeee** [comment]
We're only syncing our instructions from https://github.com/flatpak/flatpak.github.io so please add these there for now

**Comment by twihno** [comment]
Apparently I'm unable to see the difference between `flatpak/flatpak.github.io` and `flathub-infra/website` in the build script. Will open another PR in the other repo

**Comment by twihno** [comment]
TL;DR: I'm stupid

I opened the build script and removed the 4 lines related to fetching the current distro yaml from the `flatpak/flatpak.github.io` repo
1. because I didn't properly read the name of the git repo and thought it would be pulling the current version from `flathub-infra/website`
2. i wanted to use my current changes

If I had properly read the build script I would have noticed that it wants to pull the distro config from `flatpak/flatpak.github.io` and that that that repo contains the "source of truth" for the setup instructions

**Comment by razzeee** [comment]
FYI I want to change the source here, but will likely do that after you've landed everything. (if I even find the time)

https://github.com/flatpak/flatpak.github.io/issues/674

**Comment by kbdharun** [comment]
> FYI I want to change the source here, but will likely do that after you've landed everything. (if I even find the time)
> 
> https://github.com/flatpak/flatpak.github.io/issues/674

And our changes has landed upstream.
